src/test/java/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParseTreeTest.java

```diff
@@ -99,10 +106,13 @@ public void textBeforeJavadocTags()
     @Test
     public void customJavadocTags()
             throws IOException {
-        final String filename = getDocPath("InputCustomJavadocTags.txt");
-        final ParseTree generatedTree = parseJavadoc(getFileContent(new File(filename)));
-        final ParseTree expectedTree = ParseTreeBuilder.treeCustomJavadocTags();
-        compareTrees(expectedTree, generatedTree);
+        final String inputFilename = getDocPath("InputCustomJavadocTags.txt");
+        final String outputFilename = getOutputDocPath("OutputCustomJavadocTags.txt");
+        final String expectedTree = getFileContent(new File(outputFilename))
+                .replaceAll("\\\\r\\\\n", "\\\\n");
+        final String generatedTree = printFileAst(new File(inputFilename))
+                .replaceAll("\\\\r\\\\n", "\\\\n");
```

Comment:
`printFileAst().replaceAll()` and `getFileContent().replaceAll()` look like pieces of code which should be extracted to private methods, as well as `"\\\\r\\\\n"` and `"\\\\n"` to constants